FirebaseFunctions.podspec

```diff
@@ -28,6 +28,9 @@ iOS SDK for Cloud Functions for Firebase.
   s.source_files = 'Functions/FirebaseFunctions/**/*'
   s.public_header_files = 'Functions/FirebaseFunctions/Public/*.h'
 
+  s.frameworks = [
+    'UIKit',
```

Comment:
Why is this needed? I see a UIKit usage in the Functions Example, but not in the library.

Reply:
No the Example code is not part of the pod. Please remove. cc: @bklimt

Reply:
Got it, thanks for clarifying. Removed.